src/extension.ts

```diff
@@ -1737,6 +1738,27 @@ const debugAttachKubernetes = async (explorerNode: ClusterExplorerResourceNode)
     }
 };
 
+const selectPodKubernetes = async () => {
+    const resource =  await kubectlUtils.getResourceAsJson<KubernetesCollection<Pod>>(kubectl, "pods");
+    if (!resource) {
```

Comment:
If there are no pods should we show a message rather than just returning with nothing?